fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -450,10 +450,18 @@ private FHIRRestOperationResponse doPatchOrUpdate(String type, String id, FHIRPa
                 return metaResponse;
             }
 
+            // Store the payload if we're offloading
+            // TODO perhaps store the newVersionNumber value in the FHIRRestOperationResponse so
+            // we don't have to keep converting to an int. At the very least we need a helper
+            // function for this.
```

Comment:
maybe open new low-priority tech debt issue so we don't drop this?

Reply:
Created a new method `FHIRPersistenceSupport.getMetaVersionId(resource)`